libevmasm/Assembly.cpp

```diff
@@ -264,7 +264,7 @@ Json::Value Assembly::assemblyJSON(StringMap const& _sourceCodes) const
 					createJsonValue("PUSH [ErrorTag]", i.location().start, i.location().end, ""));
 			else
 				collection.append(
-					createJsonValue("PUSH [tag]", i.location().start, i.location().end, string(i.data())));
```

Comment:
Oh shoot - this was modified recently?

Reply:
Ah sorry, confused it with the one below. What is the difference between the two versions?

Reply:
There's one we use (PushTag) and there's the (PushString) which is only used by LLL right now.